### DISCUSSION TITLE
Dates

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#382] [General]
Issue came up in review of [008 mapping](https://docs.google.com/spreadsheets/d/14GtYCqPEDPx34FEcy0qxsJ5ItRPXUutPFRUiNsNW3WU/edit?usp=sharing):
Character position 06 indicates meanings of Date 1 and Date 2, which often repeat information in 26Z $c in a structured way that computers can interpret more reliably than 26X $c which includes all sorts of stuff.
* Does this data belong in RDA?
* Can we afford to lose it?
* If we can't afford to lose it, how will we map it?
    * Options include reification of statements, minting timespans, appending boilerplate to dates, specific properties
This discussion should take 26X $c, 045, and 046 into consideration as well.

**Comment by GordonDunsire** [comment]
Yes, this data belongs in RDA.

Can we afford to lose it? Depends ...

How will we map it?

The values of $c in 260, 264 may not be timespans; e.g.

"1967, c1965"
"1968 [i.e. 1971]"
"<1981=>" // ???
"©2002"
"copyright 2005"

These MARC 21 Bibliographic examples indicate how difficult it will be to parse out a timespan from $c (sometimes).

So where $c is not a simple, obvious date, it is better to use 008 Date 1 and Date 2 and ignore $c.

"u" values can be simply converted to ISO 8601; e.g. "19uu" > "19"; "195u" > "195".

The value of 008/06 is also useful for parsing a value in $c:

"b": convert the value of $c to a standard BCE value such as ISO 8601.
"c": add "-" to value of Date 1 and use for 'has date of manifestation' (use "/" if ISO 8601 is used?).
"d": concatenate Date 1 and Date 2 separated by "/" (the ISO 8601 interval designator) ("--" can also be used). This is the equivalent of converting $c by replacing "-" with "/".
"e": concatenate Date 1 and Date 2 separated by "-" (the ISO 8601 date part separator), remove "#" and "u" characters from Date 2, and insert "-" in Date 2 between the second and third characters if applicable. E.g. "1983-06-15"; "1977-05", "1976-11"
"i": same as "d". Note that this is easier than attempting to parse $c "18--?-1890".
"k": same as "d".
"m": same as "d".
"n": not applicable.
"p": same as "d".
"q": same as "d".
"r": use Date 1 only for 'has date of manifestation'. The value of Date 2 can be transformed to a note: "Date of original manifestation: {Date 2}.".
"s": use Date 1 only.
"t": use Date 1 only for 'has date of manifestation'. The value of Date 2 is mapped to 'has copyright date'.
$u: same as "c".

**Reply by CECSpecialistI** [reply]
Thank you for this @GordonDunsire ! Will be useful in guiding our continued work on the 008 mapping. This comment brings up another interesting question about dates--we've been using LC's [EDTF format](https://www.loc.gov/standards/datetime/) in MARC records in the U.S. Should we be using the [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) for our transform instead? Just when there is no $2 edtf? Always?

**Reply by CECSpecialistI** [reply]
EDTF is an extension of the ISO. Registered with ISO, so approximate dates with question marks etc. could be used

**Reply by CECSpecialistI** [reply]
Questionable/uncertain dates shouldn't be recorded...they aren't accurate. How do we map such things?

Replace with broader Wikidata IRI's for decades as timespans?

**Reply by CECSpecialistI** [reply]
EDTF free, ISO $$

**Reply by CECSpecialistI** [reply]
attn: @gerontakos @pan-zhuo

**Reply by CECSpecialistI** [reply]
use EDTF

**Comment by CECSpecialistI** [comment]
Related discussion to start @CECSpecialistI --> To what extent do we reuse systematic IRI's for places/timespans/etc. in this mapping?

**Reply by CECSpecialistI** [reply]
Not for 008 at least.

**Comment by CECSpecialistI** [comment]
Another question came up during the meeting today. For open-ended dates, such as a publication date of 1995-, can we record this in RDA as a timespan if the definition is "a finite period of time"? Can "1995-" be recorded as a value for properties which can use either a Timespan or "as an appellation of Timespan"? Is it possible that we will need to address this during Phase I for some types of single expression or augmentation aggregate manifestation? Are multipart works published over a period of time aggregates? Dickens novels in serialized form, successive novels, encyclopedias, excerpts? When does a part of a single work make something an aggregate?  @tmqdeborah @GordonDunsire

**Reply by GordonDunsire** [reply]
@CECSpecialistI: The LRM definition of Time-span is 'A temporal extent having a beginning, an end and a duration', so it must have an end, and '1975-' isn't an RDA Timespan (no end, might be infinite).

**Reply by CECSpecialistI** [reply]
Thank you Gordon! In that case, it seems that open-ended dates are not valid values for properties which have a range of Timespan even when an option exists to record as an appellation of Timespan, because they are not valid appellations of RDA timespans. Do these need to simply be notes? How does this impact RDA's ability to describe ongoing diachronic works or integrating works which have not yet concluded?

**Reply by GordonDunsire** [reply]
@CECSpecialistI: [I haven't checked RDA or previous discussions on this topic so I hope I don't contradict myself or RDA too much. I'm sure @tmqdeborah will correct me ;-)

For a manifestation of an ongoing cumulative/successive diachronic work, the first date of an open-ended date range is best recorded as a timespan with the description of the relevant issue manifestation, with a note on manifestation added to the (incomplete) description of the 'diachronic' manifestation. A relevant manifestation statement about the first date of the range can also be added to the descriptions of the diachronic manifestation and/or the issue manifestation. Additional detail may be given in a chronological designation. I guess the permissiveness of unstructured descriptions will accommodate 'broken' timespans, but for machine actionability they must be treated as notes/annotations at 'global' level. ***

When the date range becomes closed, for whatever reason, the timespan can be identified as a related entity with a separate description. One reason to do this is for data provenance, such as timespan of validity - in the context of diachronic works this is amplified by the WEM-lock. Whatever, the date range timespan accommodates the start date timespan and the end date timespan, which are ideal for machine processing to support multiple end-user facilities that are difficult or impossible to achieve outside of the LRM/RDA universe.

RDA treats iterative diachronic works in a similar way; I forget the details :-) 

*** This is the case for all 'broken' timespans, including 'circa' and 'questionable' values from the archaeology of AACR that have been a bugbear for systems librarians for the last 50 years [rant over]. In principle, all RDA unstructured descriptions are 'local' with no means for a 'global' semantic web to interpret/reason with except as generic strings that can be processed for keyword extraction, etc. It is possible, of course, to provide data provenance for the 'local' source of the string value that may be linked to a string encoding scheme; this would help other 'local' users of the data to interpret it in a semantic context. But this just shifts the burden of interpretation from the source to the consumer. In many cases keyword extraction and/or display as a note is all that is required. To accommodate additional machine processing of chronological data, the source should transform '1975-' to a minted URI for the manifestation of an issue of the diachronic work and add '1975' as its date of publication, as well as adding a note such as 'Date of publication: 1975-' to the MARC 21 manifestation that is being described. Do not panic! The hitch-hiker's guide to the (bibliographic linked open data) universe updates itself and can always be relied upon :-)

**Reply by CECSpecialistI** [reply]
Thank you @GordonDunsire ! We will discuss this at the upcoming meeting and integrate it into our mappings and transformation code. Do you mind if I quote your response here in our Phase I wrap-up documentation?

**Reply by GordonDunsire** [reply]
@CECSpecialistI: This can be used in the wrap-up documentation, but only if it is double-checked in general, and in particular where I mention uncertainty because I haven't checked myself :-)

However, I think the minted URI for the manifestation of an issue of a diachronic work requires careful consideration. To minimize global duplication, the URI should be based on an AAP for manifestation that uses a distinct SES that bridges the diachronic work/expression/manifestation that is being described with the chronological or numeric designation of the issue. A change in designation does not indicate a WEM-lock boundary.

I think there may be some CONSER or ISSN activity out there that might shed light.

**Comment by corialanus** [comment]
For a serial work the extension plan is 'successive indeterminate' and the mode of issuance is 'multiple unit'. This is already indicating that there will be more of these things that just the one you may have 'in hand'.

For date of publication which is what the 1975- in this example represents the definition is: "A timespan during which a published manifestation is published, released, or issued." So we have the beginning of the timespan of when this work has started to be issued - 1975. We don't have the end of the timespan yet because it is still 'active'.

NLNZ is recording this element as an unstructured description.

At this option https://access.rdatoolkit.org/en-US_ala-26b84eaa-054e-3c8c-8933-8760b9b2046f/div_lc4_gjt_bhb  NLNZ PS tells me "For serial works determine the appropriate value(s) for this element, then go to NLNZ SES Manifestation: date of production | date of publication for string encoding scheme instructions."

The SES tells me to add the hyphen for an open ended date
**Start date only**
Record the start date followed by a hyphen. Do not record any other punctuation.
Examples
1999‐
[1956 or 1957?]‐

So I can record an unstructured description as I have in the past. 

My NLNZ encoding document tells me:
Manifestation > date of publication
MARC bibliographic
Unstructured description
264 #1 ‡c

**Comment by tmqdeborah** [comment]
After a short converstation with @GordonDunsire, my interpreation of his message is below. Please correct me, Gordon, if I am wrong in this interpretation.

- For a manifestation of an ongoing diachronic work (successive (including 'multipart monographs' issued over time) or integrating), 
  - "the first date of an open-ended date range is best recorded as a timespan with the description of the relevant issue manifestation", i.e.:
    - describe the manifestation of the first issue (static work)
      - record as Date of manifiestation: Timespan: 1975 
      	(in most cases, we will not have an appellation for the first issue; so I would not recommend describing it in a transform)

  - "with a note on manifestation added to the (incomplete) description of the 'diachronic' manifestation ... Additional detail may be given in a chronological designation", i.e.:
    - describe the manifestation of the diachronic work:
      - record as Note on manifestation: Date of manifestation: 1975- 
      - or as Date of manifestation: 1975/ (unstructed description; Gordon says that an unstructured description is just like a note); use the ISO 8601 interval designator ("/") instead of a hyphen ("- ")
    - also (optional) record in the description of the manifestation of the diachronic work:
      - First chronological designation of sequence: 1975 
        - range is Nomen; so we could provide scheme of nomen: ISO 8601

  - Extrapolating, if/when the closing date is provided:
    - record in the description of the manifestation of the diachronic work:
      - Date of manifestation: 1975/2015 (structured)
      - and/or Date of manifestation: 1975/2015 Timestamp IRI
      - and (optional) Last chronological designation of sequence: 2015
        
We cannot use a Timespan IRI for broken timespans, e.g., "circa", "approximately",  "questionable", "between" and incomplete dates (e.g., " 20--") but we can record these as unstructured descriptions.

But the question is whether we should use such unstructured descriptions as date qualifiers in an AP for a manifestation?

**Reply by tmqdeborah** [reply]
I did some searching on our 5.6mil LC records from 2022 and found 19092 record(s) that matched the pattern: 008/07= blank OR 008/07= u OR 008/07= -

    Examples
    008 000411muuuu9999
    008 810403nuuuuuuuu
    008 000721s   
    008 000615s199u       
    008 000321m19uu1998 
    008 000317n        
    008 000515b        
    008 001102|                  
    
Here are samples of the 26X$c values found in those records
$c-1824.
$c-1957]
$c-1962?]
$c-<2004>.
$c-[1912]
$c-[2001?]
$c-c1931.
$cÂ1999
$cp1999.
$ccp2000.
$c[198-?], p1979.
$c188-?]
$c19--]
$c22 cm.
$c760 [1999 or 2000]
$c760-<763> [1999 or 2000-<2002 or 2003>]
$c<      -c1998>
$c<  -1999>
$c<   >-c1990.
$c<198-?-199-?>
$c<1996-     >
$c<1999-2004>
$cApril 22, 1799.
$cApril, 1795.
$cC2000.
$cCopyright 1900
$c<1976-1995    (1991-1994 printing)>
$cHeisei 12 [2000]
$cHeisei 12-<23> [2000-<2011>
$cJuly, A.D. 1796.
$c[between 1814 and 1820]
$c[ca. 1850]
$c[not before 1856]
$c[not after 1856]
$canno 1574.
$cp1999, 1959.
$cp1999, 1979-80.
$cthe yere of Our Lorde 1495 ...
$c[n.d.]
$c[c1976 i.e. c1978]

Picture these as qualifiers for access points for manifestations (and then as part of an IRIstring):
 
    Case book on the law of delict (19uu : Juta & Co. : volume) -- from 008 Date1
    Case book on the law of delict (19-- : Juta & Co. : volume) -- from 26X$c

    Case book on the law of delict (185u : Juta & Co. : volume) -- from 008 Date1
    Case book on the law of delict (185-? : Juta & Co. : volume) -- from 26X$c

    Case book on the law of delict (198u/1933 : Juta & Co. : volume) -- from 008 Date1
    Case book on the law of delict (between 1988 and 1992 : Juta & Co. : volume) -- from 26X$c

Is it better to include them (to prevent merging descriptions of different manifestations because someone entered different 26X$c data), or not include them (to prevent creating different descriptions of the same manifestations, because someone entered different 26X$c data)?

If we do decide to include 26X$c data, should we include all data provided or just clean dates (if it is even possible to pull out just clean dates)?

**Reply by GordonDunsire** [reply]
The difference between a 'note on manifestation' with an introductory phrase and an unstructured description of a specific element is that special instructions do not have to be developed to process data in unstructured descriptions. A note is a note (processed for keyword extraction and general display). An unstructured description a) requires identification as such and b) requires analysis, discussion, community agreement, etc., on anything other than processing for keyword extraction and general display. The main benefit of an unstructured description is its direct association with other recording methods for the same element (good for display) but this is also a major dis-benefit for the utility of a structured description or associated identifier or IRI (bad for access).

I think there is a general shift of balance between display and access in metadata standards, so the benefit of an unstructured description is declining while the dis-benefit is increasing. This may change with AI, who knows?

The utility of a structured date as a qualifier of an AAP is obvious for Manifestation. The utility of an unstructured date as a qualifier for an AAP depends on meeting the a) and b) requirements.

For b), in my opinion the best the transform can do is extract the first year date from the unstructured string and use that as the qualifier.  I don't think any arguments about the 'access point' are relevant; the question is whether the minted IRI reduces duplication or mis-merging:

casebookonthelawofdelict198u1933jutacovolume
vs
casebookonthelawofdelict1988jutacovolume
